tardis/model/base.py

```diff
@@ -72,7 +72,14 @@ def __init__(self, velocity, homologous_density, abundance, time_explosion,
         self.v_boundary_outer = v_boundary_outer
         self.homologous_density = homologous_density
         self._abundance = abundance
+        self.decayed = False
         self.time_explosion = time_explosion
+
+        self.raw_abundance = self._abundance
```

Comment:
@wkerzendorf I have now used `cProfile` module, If you want to know , no. of function calls and time taken for decay method.
  
- For 10 elements
6405 function calls (6351 primitive calls) in 0.007 seconds

- For 20 elements
7635 function calls (7581 primitive calls) in 0.008 seconds

- For 50 elements
11325 function calls (11271 primitive calls) in 0.011 seconds

- For 100 elements
17475 function calls (17421 primitive calls) in 0.016 seconds